lib/gui/app.js

```diff
@@ -154,6 +155,17 @@ app.run(() => {
         allowSleepUpdateCheck: currentReleaseType === release.RELEASE_TYPE.PRODUCTION
       });
     });
+
+  // If the error is an update user error, then we don't want
+  // to bother users each time they open the app.
+  // See: https://github.com/resin-io/etcher/issues/1525
+  }).catch((error) => {
+    return errors.isUserError(error) && error.code === 'UPDATE_USER_ERROR';
+  }, (error) => {
+    analytics.logEvent('Update check user error', {
```

Comment:
I guess in the majority of cases, if we get an error while trying to download the updates list from S3 (and it fails due to network issues), then chances are that the error we're trying to log here won't be able to make it back to Mixpanel either ;-)